app/controllers/group_controller.rb

```diff
@@ -0,0 +1,34 @@
+class GroupController < ApplicationController
+
+  def new
```

Comment:
When you have empty actions in your controllers you can remove them. If a view is created for that action, rails will find it and display it